types/validation_test.go

```diff
@@ -184,8 +195,10 @@ func TestValidatorSet_VerifyCommitLight_ReturnsAsSoonAsMajorityOfVotingPowerSign
 	vote.ExtensionSignature = v.ExtensionSignature
 	commit.Signatures[3] = vote.CommitSig()
 
-	err = valSet.VerifyCommitLight(chainID, blockID, h, commit)
+	err = valSet.VerifyCommitLight(chainID, blockID, h, commit, false)
 	assert.NoError(t, err)
+	err = valSet.VerifyCommitLight(chainID, blockID, h, commit, true)
+	assert.Error(t, err) // counting all signatures detects the malleated signature
 }
 
 func TestValidatorSet_VerifyCommitLightTrusting_ReturnsAsSoonAsTrustLevelOfVotingPowerSigned(t *testing.T) {
```

Comment:
Maybe rename the test here because now it contains a test case where we do not `ReturnAsSoonAsTrustLevelOfVotingPowerSigned`.